src/App.js

```diff
@@ -1,26 +1,24 @@
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
 import Plus from './templates/add.svg';
 
 import Title from './components/Title';
 import Input from './components/Input';
 import Button from './components/Button';
 import List from './components/List';
 import Counter from './components/Counter';
+import { addTask, removeTask } from './store/actions/listActions';
 
 class App extends Component {
   state = {
-    tasks: ['something'],
     value: '',
   }
-
-  addTask = () => this.setState(( prevState ) => Boolean(prevState.value.trimRight()) && !prevState.tasks.includes(prevState.value) ? prevState.tasks.push(prevState.value) : null)
-
-  removeTask = idToRemove => this.setState(( prevState ) => ({tasks: prevState.tasks.filter((task, id) => id !== idToRemove )}))
+  addTask = () => Boolean(this.state.value.trimRight()) && !this.props.tasks.includes(this.state.value) ? this.props.addTask(this.state.value) : null
```

Comment:
`this.state.value` is used 3 times in this function, we should destructure it. Also, better and safer check would be `isEmpty` from lodash to check if there is some values instead of Boolean constructor.